src/client/__init__.py

```diff
@@ -479,22 +479,22 @@ def aggregate_solutions_by_task(self, request: aggregation.WeightedDynamicOverla
     def find_aggregated_solutions(self, operation_id: str, request: search_requests.AggregatedSolutionSearchRequest,
                                   sort: Union[List[str], search_requests.AggregatedSolutionSortItems, None] = None,
                                   limit: Optional[int] = None) -> search_results.AggregatedSolutionSearchResult:
-        """Gets aggregated responses from Toloka that match certain criteria.
+        """Finds aggregated responses that match certain criteria.
 
         Pass to the `find_aggregated_solutions` the ID of the operation started by the [aggregate_solutions_by_pool](toloka.client.TolokaClient.aggregate_solutions_by_pool.md) method.
 
-        The number of returned aggregated responses is limited. To find remaining matching responses, call `find_aggregated_solutions` with updated filter criteria.
+        The number of returned aggregated responses is limited. Find remaining matching responses with subsequent `find_aggregated_solutions` calls.
```

Comment:
Now it is not clear that subsequent calls should be made using updated search request: in these methods user usually need to find the latest timestamp in some datetime field and make the next request filtering out all answers having less timestamp in such field.